pkg/utils/utils_test.go

```diff
@@ -83,3 +83,25 @@ func TestContains(t *testing.T) {
 		t.Errorf("expected: %v, got: %v", expected, got)
 	}
 }
+
+func TestRemoveHypelink(t *testing.T) {
+
+	type test struct {
+		input    string
+		expected string
+	}
+
+	tests := []test{
+		{input: "get <http://prometheuses.monitoring.coreos.com|prometheuses.monitoring.coreos.com> --cluster-name <http://xyz.alpha-sense.org|xyz.alpha-sense.org>", expected: "get prometheuses.monitoring.coreos.com --cluster-name xyz.alpha-sense.org"},
+		{input: "get <http://prometheuses.monitoring.coreos.com|prometheuses.monitoring.coreos.com>", expected: "get prometheuses.monitoring.coreos.com"},
```

Comment:
Please add a testcase with hyperlinks for both - resource and cluster name

Reply:
first case is for resource and clustername